doc/design/logs.md

```diff
@@ -0,0 +1,35 @@
+# Logs in SQLFlow
+## Motivation
+In order to know well about the runtime status of the SQLFlow job, we need to count the number of `TRAIN/PREDICT/EXPLAIN/NORMAL` tasks over a period of time. For example, using ELK Stack for log query and analysis‎. Generally, such statistics components are implemented by logs.
+
+## logging library
+Several statistics components use `logtail` to collect logs. The `logtail` demands a file. If such information is written to `stdout`, we should redirect the `stdout` to a local file, the file size will increase until the SQLFlow server terminates, so we need a rolling file to store the log. Besides, a well-formatted log is readable.
```

Comment:
I am not sure what is a rolling file. Is it a fixed size file that contains the most recent logs?

Reply:
A rolling file used to log rotation, and yes, it is a fixed size file to restrict the volume of the log data and to avoid overflowing the record store and keeping the log files small enough so viewers can still open them.